hooks/playbooks/control_plane_ceph_backends.yml

```diff
@@ -6,15 +6,12 @@
     - name: Load vars from previous hooks
       when: cifmw_control_plane_ceph_backend_include_vars is defined
       ansible.builtin.include_vars:
-        file: "{{ item }}"
-      loop: "{{ cifmw_control_plane_ceph_backend_include_vars }}"
+        file: "{{ cifmw_control_plane_ceph_backend_include_vars }}"
```

Comment:
I went to look this var afterwards, and it is a list[1], but I wasn't able to stop Tide to merge.
We also need to fix it here: https://github.com/openstack-k8s-operators/ci-framework/blob/afaa083d2b02b39f15264a59152d932e83bf9408/scenarios/centos-9/ceph_backends.yml#L2-L3